src/SharedType.elm

```diff
@@ -15,3 +15,8 @@ type Tabs
 type KodiTabs
     = Audio
     | Video
+
+
+type Selected
+    = Title
```

Comment:
Should probably be `Title SortDirection` and similar for all other types here.

```
type SortDirection
    = Asc
    | Desc
    | None
```

Reply:
I don't think we need a `None` in this because if any of the buttons is not clicked then the album_list is already in its normal order and if a button is clicked then according to it the album list will be sorted irrespective of the fact that what is the current state (Asc/Desc) of previously clicked button.  

**For ex:** We clicked on the` Title button `and it sorted the album list in Asc order according to title and after that, we clicked on `Date Added`, then the albums visible will appear on the front-end sorted according to `Date Added `.

Reply:
Your correct, I just checked the old interface. I still would recommend to do a custom type, as bool get messy in my opinion.

Reply:
Yes, defining a custom type is a better option. I will do that :slightly_smiling_face: